src/Analyzers/Core/Workspaces/WorkspacesAnalyzers.projitems

```diff
@@ -0,0 +1,16 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <MSBuildAllProjects>$(MSBuildAllProjects);$(MSBuildThisFileFullPath)</MSBuildAllProjects>
+    <HasSharedItems>true</HasSharedItems>
+    <SharedGUID>ec946164-1e17-410b-b7d9-7de7e6268d63</SharedGUID>
+  </PropertyGroup>
+  <PropertyGroup Label="Configuration">
+    <Import_RootNamespace>Microsoft.CodeAnalysis.Analyzers</Import_RootNamespace>
+  </PropertyGroup>
+  <ItemGroup>
+    <Compile Include="$(MSBuildThisFileDirectory)DiagnosticAnalyzerCategory.cs" />
```

Comment:
You shouldn't need this new project. You can directly move these 3 files from `Analyzers.shproj` to `CompilerExtensions.shproj`, latter will make these types available everywhere in CodeStyle layer as well as Workspace and above in other parts of Roslyn.

Reply:
I see.